### PR TITLE
Add documentation for API keys management API

### PR DESCRIPTION
**Pull request by juliendbg**

### CODE REVIEW

File: source/includes/_api_keys.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
+# API keys
+
+API keys are randomly generated passwords that can be used as an authentication method to access a protected API endpoint as an authorized user.
```
Comment (vbarbaresi) [inline source/includes/_api_keys.md:3]: authorized and authenticated, technically

**Review by vbarbaresi — APPROVED** [review]

**Comment by vbarbaresi** [comment]
Where can we find the list of permissions (as string for API use, like `edit_dataset`) in the doc?

File: source/includes/_api_keys.md (hunk 1 above)
Comment (juliendbg) [reply source/includes/_api_keys.md:3]: What do you mean? The API key is an authentication method to ensure the user has the required authorization to access the endpoint, no?

**Comment by juliendbg** [comment]
I still have to do that, plus saying which permission grants which access, it will follow in another pull request.

**Review by meganelacheny — APPROVED** [review]

**Pull request merged**